app/lib/facets_from_facet_group_extractor.rb

```diff
@@ -0,0 +1,36 @@
+class FacetsFromFacetGroupExtractor
+  def initialize(content_item)
+    @facets = content_item['links']['facet_group'].first['links']['facets']
```

Comment:
I know we test for the existence of facet links in the calling `FacetExtractor` class but this line makes me wonder whether the two are correctly coupled. This constructor makes an assumption about the data it's passed, wouldn't it be better to let the class initialisation handle missing data.

Even a more guarded
```ruby
@facets = content_item['links'].fetch('facet_group', []).first.fetch('links', {})['facets']
```
might be less brittle.

Also I think when we're at the point where we're chaining so much data lookup there's an argument for handling only the data we need, is it possible to pass only `content_item['links']['facet_group']` to the constructor?